OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs

```diff
@@ -455,6 +455,21 @@ public override void Draw()
 				{
 					var first = icon.Queued[0];
 					var waiting = first != CurrentQueue.CurrentItem() && !first.Done;
+					var multiplier = ProductionItem.TimeFactor;
+
+					if (CurrentQueue.Info.ParallelBuild)
+					{
+						waiting = false;
+						var progressingQueue = new List<ProductionItem>();
+
+						foreach (var q in CurrentQueue.AllQueued())
+							if (progressingQueue.Find(i => i.Item == q.Item) == null && !q.Paused && !q.Done)
+								progressingQueue.Add(q);
+
+						if (progressingQueue.Count > 0)
```

Comment:
I think the convention in the current code would be to use processingQueue.Any()